### PR TITLE
Use `state.title` in more places

### PR DESCRIPTION
**Pull request by bmintz**
See #689

### CODE REVIEW

File: app/fileManager.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default function(state, emitter) {
     document.addEventListener('blur', () => (updateTitle = true));
     document.addEventListener('focus', () => {
       updateTitle = false;
-      emitter.emit('DOMTitleChange', 'Firefox Send');
+      emitter.emit('DOMTitleChange', state.title);
```
Comment (dannycoates) [inline app/fileManager.js]: This isn't what we need here. The `DOMTitleChange` event sets `state.title` to the passed argument, so this effectively does nothing. A good way to do this is to add a config parameter to the server and pass it through to the frontend.

**Comment by dannycoates** [comment]
Accomplishing #689 needs more work than this, and unfortunately this patch doesn't really help towards it. Factoring out the product name is not a priority for us right now, but we'd consider a more complete patch from the community.

File: app/fileManager.js (hunk 1 above)
Comment (bmintz) [reply app/fileManager.js]: Could you elaborate on that alternative?

Comment (dannycoates) [reply app/fileManager.js]: We'll probably need:
- a new config parameter / variable in `server/config.js` for `PRODUCT_NAME`
- add said variable to `server/routes/jsconfig.js`
- add that variable to `state` in `app/main.js`
- also add it to `server/state.js`
- change the hard coded strings

**Pull request closed without merging**